### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by eric-hansen** [boone-studios/laravel-surrealdb#17] [General]
I've been looking into migrating from MariaDB over to Surreal for a bit, and right now my main ecosystem is Laravel.  So before doing such a thing I would need to either: A) rewrite my projects in something that's compatible w/ Surreal (i.e.: in Rust) or B) have a DB driver in Laravel for Surreal.

I like the syntax and such of this package, but the last commit (on main at least) is from May 22nd.

Is this still being actively developed?  If so what help is really needed?  I did look at the issues tab, and most seem pretty logical.  But I'm not sure where the progress is on most things, if they've been started on.

**Comment by leemcd56** [comment]
It's still alive. I've been tied up with work, and haven't been able to focus on it 100%. You're more than welcome to help me out with this package, and add any features that are needed!

I do have some un-pushed commits, but the code isn't complete. I've been hoping for contributions from the community to help move this package forward.

**Reply by eric-hansen** [reply]
Sure, I'll help where I can.

I've never written a DB driver for Laravel before, so I may have some questions along the way but I'll pick up some of the smaller things first to get a feel for stuff.

**Reply by adaliszk** [reply]
Same here. I also have not written any package for Laravel yet, but I do have extensive plans utilizing SurrealDB with Laravel, so I will start contributing soon.
